packages/block-editor/src/components/provider/index.js

```diff
@@ -1,139 +1,88 @@
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
-import { withDispatch } from '@wordpress/data';
-import { compose } from '@wordpress/compose';
+import { useEffect, useRef } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
```

Comment:
Nitpick: I think imports should be in alphabetical order.

```suggestion
import { useDispatch, useSelect } from '@wordpress/data';
import { useEffect, useRef } from '@wordpress/element';
```